RESTFulSense/Services/Orchestrations/Forms/FormOrchestrationService.cs

```diff
@@ -74,12 +76,62 @@ private void AddStringContents(FormModel formModel)
 
                 if (attribute != null)
                 {
-                    string value = (string)this.valueService.RetrievePropertyValue(formModel.Object, property);
-                    this.formService.AddStringContent(formModel.MultipartFormDataContent, value, attribute.Name);
+                    bool ignoreDefaultValues = attribute.IgnoreDefaultValues;
+                    object propertyValue = this.valueService.RetrievePropertyValue(formModel.Object, property);
+                    bool isDefaultValue = IsDefaultValue(property.PropertyType, propertyValue);
+
+                    if (isDefaultValue is false || ignoreDefaultValues is false)
+                    {
+                        string value = ConvertToString(propertyValue, attribute.MediaType, ignoreDefaultValues);
+                        this.formService.AddStringContent(formModel.MultipartFormDataContent, value, attribute.Name);
+                    }
                 }
             }
         }
 
+        private static bool IsDefaultValue(Type type, object value)
+        {
+            if (type.IsValueType)
+            {
+                object defaultValue = Activator.CreateInstance(type);
+
+                return value.Equals(defaultValue);
+            }
+            else
+            {
+                return value is null;
+            }
+        }
+
+        private static string ConvertToString(object propertyValue, string mediaType, bool ignoreDefaultValues)
+        {
+            return mediaType switch
+            {
+                "text/json" => ConvertToJsonString(propertyValue, ignoreDefaultValues),
+                "application/json" => ConvertToJsonString(propertyValue, ignoreDefaultValues),
+                _ => propertyValue.ToString()
```

Comment:
We should guard for null `propertyValue`. This code path is triggered when `property` has a default value of null and we don't ignore default values.

```suggestion
                _ => propertyValue?.ToString()
```